compiler/include/passes.h

```diff
@@ -120,6 +120,9 @@ void fixupErrorHandlingExits(BlockStmt* body, bool& adjustCaller);
 void addDummyErrorArgumentToCall(CallExpr* call);
 bool isVirtualIterator(Symbol* iterator);
 
+// LoopExpr.cpp
+void removeDeadTypeStuff(BlockStmt* block);
```

Comment:
Let's not add this here and let it remain in functionResolution.cpp as a static function